client/src/Movies/Movie.js

```diff
@@ -26,42 +30,31 @@ export default class Movie extends Component {
       });
   };
   // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  componentDidUpdate(newProps) {
```

Comment:
good. componentWillReceiveProps I believe will be deprecated in the future. Good to update that with componentDidUpdate